### DISCUSSION TITLE
how to change the font size of pages under `docs` dir?

### DISCUSSION THREAD

**Discussion by jackywu** [web-infra-dev/rspress#948] [Q&A]
the default font size is too large. how can I change it ?

**Comment by Timeless0911** [comment]
The default `font-size` is 16px in rspress, https://github.com/web-infra-dev/rspress/blob/3a8251f61dc1332f47ac4027e83cd3f7b401c9b2/packages/theme-default/src/styles/base.css#L27

You can use custom styles to override it https://rspress.dev/guide/basic/custom-page.html#custom-styles
